tests/stp_test.py

```diff
@@ -0,0 +1,407 @@
+import os
+import pytest
+from click.testing import CliRunner
+
+import config.main as config
+import show.main as show
+from utilities_common.db import Db
+from .mock_tables import dbconnector
+
+
+show_spanning_tree = """\
```

Comment:
Looks like these are expected result of a single unit test. I suggest:
1. Capitalize the variable, since they are constant.
2. If they are only used by one unit test, just define it in the unit test
3. be more specific in defining variable name, such as "EXPECTED_SHOW_SPANNING_TREE_OUTPUT".

These types of assertion also create lots of churns when changing the code. A single newline or space introduced in the future can fail a test. Consider using `in` to assert something more flexible. For example, asserting the output contains a line "Spanning-tree Mode: PVST" or a table that looks like 
```
Port             Prio Path      Port Uplink State         Designated  Designated       Designated
Name             rity Cost      Fast Fast                 Cost        Root             Bridge
Ethernet4        128  200       N    N      FORWARDING    400         0064b86a97e24e9c 806480a235f281ec
```
which is easier to change in the future.